http_cache/expiration.rst

```diff
@@ -85,5 +85,12 @@ the lifetime calculation vulnerable to clock skew. Another limitation
 of the ``Expires`` header is that the specification states that "HTTP/1.1
 servers should not send ``Expires`` dates more than one year in the future."
 
+.. note::
+
+    Accordingly with `RFC 7234 - Caching`_, the ``Expires`` header value is
+    ignored when a ``s-max-age`` or ``max-age`` header is defined.
```

Comment:
In fact, `s-maxage` (note the missing second dash) and `max-age` are not headers, but are directives of the `Cache-Control` header.